servers/src/main/java/tachyon/master/journal/JournalWriter.java

```diff
@@ -29,6 +30,7 @@
 import tachyon.exception.ExceptionMessage;
 import tachyon.master.MasterContext;
 import tachyon.underfs.UnderFileSystem;
+import tachyon.underfs.s3.S3OutputStream;
```

Comment:
Perhaps try to get this information with some other kind of check (ie. UnderFileSystem Address is s3)? We want to avoid depending on S3 underfs if at all possible.